src/nnue/nnue_feature_transformer.h

```diff
@@ -345,7 +345,8 @@ class FeatureTransformer {
             constexpr IndexType NumOutputChunks = HalfDimensions / 2 / OutputChunkSize;
 
             const vec_t Zero = vec_zero();
-            const vec_t One  = vec_set_16(127 * 2);
+            const vec_t One  = vec_set_16(127 * 2); // Read more about why the "*2" is necessary here:
+                  // https://github.com/official-stockfish/Stockfish/pull/5282#issuecomment-2131582943
```

Comment:
Explaining the direct reason here would be better rather than just putting up a link.

Reply:
This was the suggestion of the patch author, it's better to link directly to the comment.